johngorithm/github-ap-with-java
# #155820624 launch detail activity from main activity

#### What does this PR do?
Launch detail activity from main activity

#### Description of tasks to be completed
- add DetailActivity activity class
- add click event on user card view
- attach a click event listener to the card view

#### How should this be manually tested?
- clone this repository
- open project using `Android Studio`
- checkout into `ft-launch-detail-activity-from-main-activity-155820624` from Android studio integrated command line
- run the app with `Run` button provided by the IDE
- click on the displayed `user` card. You should see the detail view

#### Any background context you want to provide?
No

#### Screenshots
##### Before Click
<img width="1440" alt="screen shot 2018-12-06 at 3 28 47 pm" src="https://user-images.githubusercontent.com/30831276/49590248-ae8b9400-f96b-11e8-9599-e016257006ac.png">

##### After Click
<img width="1440" alt="screen shot 2018-12-06 at 3 28 55 pm" src="https://user-images.githubusercontent.com/30831276/49590273-c19e6400-f96b-11e8-9c5a-af798ead6267.png">

#### What are the relevant Pivotal Tracker Stories?
[155820624](https://www.pivotaltracker.com/story/show/155820624)
